### DISCUSSION TITLE
How to include clustering for Circle, Polygon on vis.gl google maps?

### DISCUSSION THREAD

**Discussion by zourdyzou** [visgl/react-google-maps#496] [Q&A]
Hey, I'm wondering if there is an example for clustering not only the marker but also including the circle, polygon or polyline in vis.gl google maps adapter? if someone know how to do this please let us have a discussion

**Comment by Leo42night** [comment]
The Google [documentation](https://developers.google.com/codelabs/maps-platform/maps-platform-101-react-js#0) states that you can find the Circle component in the repository at ./src/component/Circle.jsx, but I couldn't find it there

**Comment by usefulthink** [comment]
We haven't yet added geometry primitives to the main library, but they're available with our examples for you to copy and use in your project. 

The documentation is referring to [the sample code repository](https://github.com/googlemaps-samples/codelab-maps-platform-101-react-js), the `Circle` component that is being talked about is available here: https://github.com/googlemaps-samples/codelab-maps-platform-101-react-js/blob/main/solution/src/components/circle.tsx, which I believe is a copy of the circle component in our examples: https://github.com/visgl/react-google-maps/tree/main/examples/geometry/src/components
